covid-19-support/src/components/Sidebar.vue

```diff
@@ -0,0 +1,221 @@
+          @day-selected="(opt) => $emit('day-selected', opt)"
+        />
+
+        <InfoPanel :infotype="'note'" :icon="'fa-info-circle'" v-if="location.currentBusiness == null || showListing">
+          {{ $t('sidebar.info-about-us') }} <a href="#" @click="$bvModal.show('about-us')">{{ $t('sidebar.info-link-text') }}</a
+          >{{ $t('sidebar.info-end-text') }}
+        </InfoPanel>
+
+        <InfoPanel :infotype="'handwash'" :icon="'fa-hands-wash'" v-if="filteredMarkers.length == 0">
+          <b class="themeFont">{{ $t('sidebar.shopsafe') }}</b>
+          <br />
+          (1) {{ $t('sidebar.stayhome') }}<br />
+          (2) {{ $t('sidebar.sixfeet') }}<br />
+          (3) {{ $t('sidebar.washhands') }}<br />
+        </InfoPanel>
+      </div>
+
+      <BusinessDetails
+        :infotype="'green'"
+        :icon="'fa-tractor'"
+        :business="location.currentBusiness"
+        v-if="location.currentBusiness != null && showListing != true"
+        @close-details="closeDetails"
+      ></BusinessDetails>
+    </div>
+
+    <results-list
+      :filteredMarkers="highlightFilteredMarkers"
+      :location="location"
+      @location-selected="passLocation"
+      v-if="showListing"
+      :selected-day="day"
+    />
+  </div>
+</template>
+
+<script>
+import BusinessDetails from './BusinessDetails.vue'
+import InfoPanel from './InfoPanel.vue'
+import ResultsList from './ResultsList.vue'
+import SearchFilters from './SearchFilters.vue'
+
+export default {
+  name: 'sidebar',
+  components: {
+    BusinessDetails,
+    InfoPanel,
+    ResultsList,
+    SearchFilters
+  },
+  data() {
+    return {
+      locationData: location,
+      showListing: this.showList
+    }
+  },
+  props: {
+    isFilterOpen: Boolean,
+    need: String,
+    day: Number,
+    filteredMarkers: Array,
+    highlightFilteredMarkers: Array,
+    location: { locValue: Number, locId: String, isSetByMap: Boolean, currentBusiness: Object },
+    showList: Boolean
+  },
+  computed: {
+    tabtitle() {
+      return this.isFilterOpen ? this.$t('sidebar.close-panel') : this.$t('sidebar.open-panel')
+    }
+  },
+  methods: {
+    closeDetails: function () {
+      this.showListing = true
+    },
+    passLocation: function (val) {
+      this.locationData = val
+      this.showListing = false
+      this.$emit('location-selected', val)
+    }
+  },
+  watch: {
+    day: function () {
+      this.locationData = null
+      this.showListing = true
+    },
+    need: function (val) {
+      this.locationData = null
+      if (val == 'none') {
+        this.showListing = false
+      } else {
+        this.showListing = true
+      }
+    },
+    location: function () {
+      this.showListing = false
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+.custom-select {
+  font-size: 0.8rem;
+  cursor: pointer;
+}
+
+#sidebar-wrapper {
+  transition: transform 0.25s ease-out;
+  transform: translateX(-100%);
```

Comment:
I suggest changing this to:
transform: translateY(100%);

This should allow the list and business details to slide up from the bottom, instead of in from the side.

Then seem my next comment on line 134